data/pigui/modules/flight-ui/indicators.lua

```diff
@@ -32,7 +32,7 @@ local function displayDirectionalMarkers()
 	local function displayDirectionalMarker(ship_space, icon, showDirection, angle)
 		local screen = Engine.ShipSpaceToScreenSpace(ship_space)
 		local coord = Vector2(screen.x, screen.y)
-		if screen.z <= 1 then
+		if screen.z <= 0 then
```

Comment:
Should this be `if screen.z <= 0 or screen.z >= 1 then` instead because the valid visible range is 0..1? Or does Lua see the z differently?

Reply:
I may have read that wrong, it just seems strange that it's only visible if it's less than one when the range is 0..1

Reply:
To preserve existing behavior, all of the "project to screen" routines negate the Z coordinate they return; coordinates in the visible Z range are negative, while coordinates behind the camera are positive. It's weird and I'd welcome some improvement in that area, as trying to massage the depth coordinate hasn't yielded good results so far.